README.md

```diff
@@ -80,11 +81,12 @@ instance with 60 GB disk has been sufficient.
 
 #### Make a routing to the VM
 
-You will need to make a URL to access the VM. On the AWS console, go to
-route 53 service, click **Hosted zones**. If you already have Hosted
-zone you can use, select it from the list. If not, create one to suit
-your needs. Once the zone is selected, click **Create Record Set** at
-the top. Choose a name you like and use the elastic ip as the value.
+You will need to make a URL for consistent access to the VM. On the AWS
```

Comment:
I'm not sure I like `consistent`. And in hindsight, I don't think I like `make a URL`. How about something like:

You will need to create a DNS record that maps to your VM.

Reply:
This is better.